docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md

```diff
@@ -0,0 +1,72 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'This guide helps you understand common Cloud Manager errors and sets you on the path forward to resolution.'
+og_description: 'This guide helps you understand common Cloud Manager errors and sets you on the path forward to resolution.'
+keywords: ['error','account limit','limit','activated', 'before you can', 'please try again', 'open a support ticket']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2019-02-22
+modified_by:
+  name: Linode
+title: "An Overview of Common Cloud Manager Errors"
+h1_title: "Understanding Common Cloud Manager Errors"
+contributor:
+  name: Linode
+---
+
+Have you ever encountered an error message while navigating the [Cloud Manager](https://cloud.linode.com) and wanted to know what caused the error and what to do next? This guide will cover some common error messages, their common causes, suggestions on your best paths forward.
```

Comment:
We probably don't need the first sentence. "This guide will cover some common error messages found in the Cloud Manager(link)..."

Also need the word "and" before "suggestions."